src/Data/RFC3339String.purs

```diff
@@ -25,19 +34,36 @@ instance showRFC3339String :: Show RFC3339String where
   show (RFC3339String s) = "(RFC3339String " <> show s <> ")"
 
 -- | Remove trailing zeros from the subsecond component.
-trim :: RFC3339String -> RFC3339String
-trim (RFC3339String s) =
+trim :: String -> String
+trim s =
   let
     withoutZulu = dropWhileEnd (eq 'Z') s
     withoutTrailingZeros = dropWhileEnd (eq '0') withoutZulu
     withoutTrailingDot = dropWhileEnd (eq '.') withoutTrailingZeros
   in
-    RFC3339String $ withoutTrailingDot <> "Z"
+    withoutTrailingDot
```

Comment:
This function is public, so changing the type and return value would be a breaking change.

Reply:
I would prefer that we kept the newtype though.

Reply:
Restored.